test/algorithms/closest_points/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Boost.Geometry
+# Copyright (c) 2024 Barend Gehrels, Amsterdam, the Netherlands.
+# Use, modification and distribution is subject to the Boost Software License,
+# Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+foreach(item IN ITEMS
+    ar_ar
+    l_ar
+    l_l
+    pl_ar
+    pl_l
+    pl_pl
+    )
+  boost_geometry_add_unit_test("algorithms_closest_points" ${item})
```

Comment:
This one is the only suffixing the name itself to the algorithms, for obvious reasons